pages/create-nft.tsx

```diff
@@ -0,0 +1,16 @@
+import type { NextPage } from "next";
+import Head from "next/head";
+
+const createNft: NextPage = function () {
+  return (
+    <div>
+      <Head>
+        <title>Create Next App</title>
```

Comment:
Fix the head content please.
"Hedera Minting" instead of "Create Next App", same for the content attribute.

Do we need to implement the `Head` component on every page? Can we specify a default and override it if necessary instead?

Reply:
Yes, can be done by adding default values in head in layout, and to override it, 
reuse it like this on the specific page.
<Head>
        <title>Overridden header.</title>
 </Head>

Also thought of creating a context for it but we cannot access react hooks in nexjs serverside.